system_test/aest_sync_SUITE.erl

```diff
@@ -144,7 +121,29 @@ new_node_joins_network(Cfg) ->
     Length = 20,
     NodeStartupTime = proplists:get_value(node_startup_time, Cfg),
 
-    setup_nodes([?OLD_NODE1, ?OLD_NODE2, ?NEW_NODE1], Cfg),
+    Compatible = "aeternity/epoch:local", %% Latest version it should be compatible with
```

Comment:
Is it intended to test `local` compatibility with `local` itself?

Reply:
For this version yes, but I liked Sebastiens solution to call it latest_released better. Just that I didn't get it when he called it latest.

Reply:
What is `latest`? A Docker image that already exist?

Reply:
@eproxus From https://github.com/aeternity/epoch/blame/89bb66e68e2058466a865645e16c0ead590e431f/docs/docker.md#L13
> Docker image is automatically build and published on [DockerHub](https://hub.docker.com/r/aeternity/epoch/). All tagged source code versions have their own docker image tag as well. Latest tagged ("stable") version is tagged with `latest`.

Reply:
@lucafavatella @ThomasArts So shouldn't we just use that version here then?

Reply:
> @lucafavatella @ThomasArts So shouldn't we just use that version here then?

@eproxus  Assuming that master is meant to be backwards compatible - that is currently the case - yes.